src/algebra/order/floor.lean

```diff
@@ -593,6 +593,13 @@ end
 
 @[simp] lemma fract_nonneg (a : α) : 0 ≤ fract a := sub_nonneg.2 $ floor_le _
 
+/-- The fractional part of `a` is positive if and only if `a ≠ ⌊a⌋`. -/
+lemma fract_pos_iff {a : α} : 0 < fract a ↔ a ≠ ⌊a⌋ :=
```

Comment:
```suggestion
lemma fract_pos : 0 < fract a ↔ a ≠ ⌊a⌋ :=
```

Reply:
OK. I have removed `fract_pos`, renamed `fract_pos_iff`to `fract_pos` and removed the redundant argument.